.gitignore

```diff
@@ -116,5 +116,5 @@ ENV/
 # PyCharm
 /.idea
 
-# Signing service daily tresholds
-daily_tresholds/
+# Signing service daily thresholds
+daily_thresholds/
```

Comment:
Finally someone did it ;)

Reply:
I did the typo in code, and forgot to fix this in `.gitignore`, someone could say something :P